bin/ksql-run-class

```diff
@@ -8,17 +8,11 @@ set -ue
 #cd -P deals with symlink from /bin to /usr/bin
 base_dir=$( cd -P "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
 
-# Log directory to use
```

Comment:
I think you should keep this, for providing a defualt log dir (/tmp/..) when LOG_DIR env has not been set.
This also makes sure ksql.log.dir is always set, otherwise the log4j properties will be ran out of the root / dir which will not work.